packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -774,11 +775,11 @@ export class BuildService {
       const url = `${clientHost}/${project.workspaceId}/${project.id}/${resource.id}/builds/${build.id}`;
       const buildLinkHTML = `[${url}](${url})`;
 
-      const commitMessage = oldBuild
+      const commitMessage = oldBuild?.id
         ? commit.message && `Commit message: ${commit.message}.`
-        : INITIAL_ONBOARDING_COMMIT_MESSAGE_BODY;
+        : FIRST_COMMIT_MESSAGE_BODY; // this message will be shown only on the first commit to the repository
 
-      const commitBody = `Amplication build # ${build.id}\n${commitMessage}\nBuild URL: ${buildLinkHTML}`;
+      const commitBody = `Amplication build # ${build.id}\n\n${commitMessage}\n\nBuild URL: ${buildLinkHTML}`;
```

Comment:
@PazYanoverr @morhag90 for commit messages, it would be better to have the amplication commit message on the first line like:
```sh
Amplication build #${build.id}   ${commitMessage}
Build URL: ${buildLinkHTML}
```

Plus empty lines are not really good on commit messages